sdk/core/core-client/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
+### Breaking Changes:
```

Comment:
I think we usually just put [BREAKING] or something in the line item instead of calling them out separately?

Reply:
Will do. I think I copied this from the Service Bus readmes - let me do a larger sampling :)